source/uwp/Renderer/lib/ActionHelpers.cpp

```diff
@@ -955,139 +1271,198 @@ namespace AdaptiveCards::Rendering::Uwp::ActionHelpers
+        if (lastPrimaryAction != nullptr)
+        {
+            ComPtr<IUIElement> createdButton;
+            RETURN_IF_FAILED(CreateActionButtonInActionSet(adaptiveCard,
+                                                           adaptiveActionSet,
+                                                           lastPrimaryAction.Get(),
+                                                           true,
+                                                           currentButtonIndex,
+                                                           actionsPanel.Get(),
+                                                           showCardsPanel.Get(),
+                                                           columnDefinitions.Get(),
+                                                           lastActionOverflowItem.Get(),
+                                                           renderContext,
+                                                           renderArgs,
+                                                           &createdButton));
+            currentButtonIndex++;
+        }
+
+        // Lastly add the overflow button itself to the action panel
+        if (overflowButton != nullptr)
+        {
+            // If using equal width columns, add another column and assign the it to the overflow button
+            if (columnDefinitions != nullptr)
+            {
+                ComPtr<IColumnDefinition> columnDefinition = XamlHelpers::CreateABIClass<IColumnDefinition>(
+                    HStringReference(RuntimeClass_Windows_UI_Xaml_Controls_ColumnDefinition));
+                RETURN_IF_FAILED(columnDefinition->put_Width({1.0, GridUnitType::GridUnitType_Star}));
+                RETURN_IF_FAILED(columnDefinitions->Append(columnDefinition.Get()));
+
+                ComPtr<IFrameworkElement> actionFrameworkElement;
+                THROW_IF_FAILED(overflowButton.As(&actionFrameworkElement));
+                THROW_IF_FAILED(gridStatics->SetColumn(actionFrameworkElement.Get(), currentButtonIndex));
+            }
+
+            // Add the overflow button to the panel
+            XamlHelpers::AppendXamlElementToPanel(overflowButton.Get(), actionsPanel.Get());
+
+            // Register the overflow button with the render context
+            ComPtr<IUIElement> overflowButtonAsUIElement;
+            overflowButton.As(&overflowButtonAsUIElement);
+
+            ComPtr<AdaptiveCards::Rendering::Uwp::AdaptiveRenderContext> contextImpl =
+                PeekInnards<AdaptiveCards::Rendering::Uwp::AdaptiveRenderContext>(renderContext);
+
+            if (adaptiveActionSet != nullptr)
+            {
+                RETURN_IF_FAILED(contextImpl->AddOverflowButton(adaptiveActionSet, overflowButtonAsUIElement.Get()));
+            }
+            else
+            {
+                RETURN_IF_FAILED(contextImpl->AddOverflowButton(adaptiveCard, overflowButtonAsUIElement.Get()));
+            }
```

Comment:
maybe

```suggestion
            RETURN_IF_FAILED(contextImpl->AddOverflowButton((adaptiveActionSet) ? adaptiveActionSet : adaptiveCard,
                                                            overflowButtonAsUIElement.Get()));
```

Reply:
Same issue here, the compiler can't figure this out.